src/main/java/seedu/progresschecker/logic/commands/AddDefaultTasksCommand.java

```diff
@@ -0,0 +1,48 @@
+package seedu.progresschecker.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import static seedu.progresschecker.model.task.MyTaskList.createTaskList;
+
+import seedu.progresschecker.logic.commands.exceptions.CommandException;
+
+/**
+ * Adds a default task list to the user's Google account.
+ */
+public class AddDefaultTasksCommand extends Command {
+
+    public static final String COMMAND_WORD = "newtasklist";
+    public static final String COMMAND_ALIAS = "nl"; // short for "new list"
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Creates a new task list. "
+            + "Parameters: "
+            + "LISTNAME "
+            + "Example: " + COMMAND_WORD + " "
+            + "CS2103 LOs";
+
+    public static final String MESSAGE_SUCCESS = "New task list added: %1$s";
+    public static final String DEFAULT_LIST_TITLE = "CS2103 LOs";
+    public static final String CREATE_FAILURE = "Failed to create task list: ";
+
+    private String listTitle;
+
+    /**
+     * Creates a AddDefaultTasksCommand to add the default task list with title {@code Sting}
+     */
+    public AddDefaultTasksCommand(String title) {
+        requireNonNull(title);
+        listTitle = title;
+    }
+
+    @Override
+    public CommandResult execute() throws CommandException {
+        try {
+            createTaskList(listTitle);
```

Comment:
I understand that you mentioned this command is incomplete so I'd just like to clarify on the behavior. Right now this statement causes some NoClassDefFoundError and/or ClassNotFoundException (I think), is this behavior intended? If yes, would you be completing this command by the end of milestone v1.3? o:

Reply:
updated